src/app/static/src/app/components/adr/SelectRelease.vue

```diff
@@ -7,9 +7,9 @@
                 value="useLatest"
                 v-model="choiceADR"
             />
-            <label for="useLatest" v-translate="'useLatest'" class="pr-1"></label>
+            <label for="useLatest" v-translate="'useLatest'" style="margin-left: 4px;"></label>
```

Comment:
padding right class wasnt working in a couple of places, we do use other bootstrap classes so wasnt sure why but just replaced the styles with some small pixel values, I think it looks just fine